.gitattributes

```diff
@@ -1 +1,3 @@
 *.stl filter=lfs diff=lfs merge=lfs -text
+
+*	text=auto
```

Comment:
what is this formatting lol also missing eof newline

Reply:
fixed